app/jobs/spree/intercom/create_user_job.rb

```diff
@@ -1,7 +1,9 @@
 class Spree::Intercom::CreateUserJob < ApplicationJob
 
   def perform(id)
-    Spree::Intercom::CreateUserService.new(id).create
+    if Spree::Config.enable_intercom
```

Comment:
I think we should move this condition in the intercome base service that we have created.